index.html

```diff
@@ -111,7 +111,13 @@
 theInput.oninput = translate;
 theOutput.onfocus = selectTarget;
 function selectTarget(event) {
-  event.target.select();
+  /*
+    this timeout ensures select() is called after any built-in click handlers
+    that could otherwise clear the selection (as occurs reliably in Chrome).
+  */
```

Comment:
Please use line comments and start with a capital letter. Otherwise, great comment.